web/app/themes/dp-timber-theme/components/molecule/Hero/hero.scss

```diff
@@ -0,0 +1,105 @@
+@use "../../SCSS/_variables/index.scss" as var;
+@use "../../SCSS/components/default";
+
+.hero {
+	@include var.displayFlex;
+	@include var.flexCenter;
+	padding: 6em 0 2em 0;
+	background-attachment: fixed;
+	@include var.background;
+	border-bottom: 0 !important;
```

Comment:
Do we need !important?

Reply:
No we don't need !important I removed it